.github/CONTRIBUTING.md

```diff
@@ -1,64 +1,65 @@
+If an issue with the bug or feature you want to report/suggest exists, comment on it with your info (bug reports) or give it a :thumbsup: (Feature Request) to show that this is important for you.
 
-### Our Responsibilities
+### Issue isn't caused by external source
+PlaceholderAPI provides a feature to have extensions (separate jar files) for placeholders. This gives it a possability that an issue you encounter is caused by said extensions or a separate plugin that uses those extensions.  
+In those cases should you report the issue to the issue tracker of the extension or plugin.
 
-Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+## Pull requests
+As an open source project are we welcoming all contributions to improve PlaceholderAPI, being it changes to its code like bug fixes or new features, or contributions to its documentation such as the Wiki or the Javadoc.
 
-Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+### Code contributions
+Any contributions to PlaceholderAPI's code should be done towards the `develop` branch. Targeting the `master` branch in your Pull request may get it closed without warning.  
+Additionally should you follow the `Google Codestyle` when changing the code and Javadoc.
 
-### Scope
+Some noteworthy points about Javadoc:
 
-This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+- `@param` tags are in a single line and any additional param tag after one don't have an empty line in between.  
+Example:  
+```java
+/**
+ * @param param1 description
+ * @param param2 description
+ */
+```
+- Tags follow the order `@param`, `@return`, `@since` and `@deprecated`
+- The `@deprecated` tag should mention the version of when the annotated object will be removed. This usually would be the next minor version.
 
-### Enforcement
+### Wiki/Javadoc contributions
+If your Pull request only targets the [wiki] or only changes the Javadoc of PlaceholderAPI (And not its actual code) should Pull requests target the `docs` branch.  
+The develop branch should only be targeted when you also change some of PlaceholderAPI's code.
```

Comment:
See a few comments above, regarding consulting lemmo.

Reply:
See my response about this...